anvil/src/winit.rs

```diff
@@ -218,6 +219,13 @@ pub fn run_winit(log: Logger) {
                 }
             }
 
+            // draw input method square if any
```

Comment:
Kinda nitpicky, but I think that `square` is not the right term here, square always has aspect ratio of 1:1, while input method popup is just any rectangle. Perhaps just call it `input method surface`?

Reply:
Will do

Reply:
fixed